spec/controllers/users/phones_controller_spec.rb

```diff
@@ -11,11 +11,24 @@
     allow(@analytics).to receive(:track_event)
   end
 
-  context 'user adds phone' do
+  context 'user adds phone' d
     it 'gives the user a form to enter a new phone number' do
       get :add
 
       expect(response).to render_template(:add)
+      expect(response.request.flash[:alert]).to be_nil
+
+    end
+
+    it 'displays error if phone number exceeds limit' do
+      user.phone_configurations.create(encrypted_phone: '4105555555')
+      user.phone_configurations.create(encrypted_phone: '4105555555')
+      user.phone_configurations.create(encrypted_phone: '4105555555')
+      user.phone_configurations.create(encrypted_phone: '4105555555')
+
+      get :add
+      expect(response).to redirect_to(account_url + '#phones')
```

Comment:
Is it possible to add an assertion that the rendered page contains the `#phones` anchor?

Reply:
yes line 33 checks the achor is there.     `it 'renders the #phone anchor when it exceeds limit' do`

Reply:
Sorry, I meant an assertion that checks if there is an element with the `#phone` id in addition to checking that the redirect contains the anchor. Something like:

```ruby
expect(response.body).to include('id="phones"')
```

Reply:
Ok can't find it in the response.body but I made this test and checked that the #phones is there. 
https://github.com/18F/identity-idp/blob/f550566853a735ae21c376a9d069b45f0ec8398d/spec/features/phone/add_phone_spec.rb#L113